docker/compile/deprecated/Dockerfile.ubuntu14.04

```diff
@@ -33,7 +33,7 @@ RUN apt-get update && apt-get -y install \
       libssl-dev \
       git \
       libtool \
-      python3-dev \
```

Comment:
python3.6 is the minimum supported version.

As the base OS is over a year past EOL, and there's no python3.6 packages, I think this should be removed, rather than leaving it in as moving to deprecated (unless there is a trivial python3.6 install - but I still don't think worth it)

Reply:
Good point. I've removed the image.